### PR TITLE
Add a new tab to Admin tests panel for Selects

### PR DESCRIPTION
**Pull request by peterjcoffman**
A demonstration of every currently-supported form of Select.

### CODE REVIEW

File: client-app/src/admin/tests/Select/SelectTestPanel.js [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+            )
+        ],
+        alignItems: 'center'
+    })
+);
+
+@HoistModel
+class LocalModel {
+    @bindable
+    Select;
```
Comment (lbwexler) [inline client-app/src/admin/tests/Select/SelectTestPanel.js]: If this represents a value it should have a name like selectValue.  These look like class names.

File: client-app/src/admin/tests/Select/SelectTestPanel.js [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+    @bindable
+    CreatableSelect;
+    @bindable
+    AsyncSelect;
+    @bindable
+    AsyncCreatableSelect;
+    @bindable
+    BigSelect;
+    @bindable
+    NumOptions = 1000;
```
Comment (lbwexler) [inline client-app/src/admin/tests/Select/SelectTestPanel.js]: We don't use this kind of casing for properties in javascript.

File: client-app/src/admin/tests/Select/SelectTestPanel.js [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+    const numOptionsName = (enableWindowed ? 'Windowed' : '') + 'NumOptions';
+    return fragment(
+        p((enableWindowed ? 'Windowed' : '') + 'Select with many options'),
+        label('value: ' + model[name]),
+        select({
+            model,
+            enableWindowed,
+            bind: name,
+            options: function() {
+                let options = [];
```
Comment (lbwexler) [inline client-app/src/admin/tests/Select/SelectTestPanel.js]: generate these in the model itself, with a reaction.

File: client-app/src/admin/tests/Select/SelectTestPanel.js [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+                    exampleAsync(model, true),
+                    exampleAsyncCreatable(model, true),
+                    exampleBig(model, true)
+                )
+            )
+        });
+    }
+});
+
+function exampleSelect(model, enableWindowed) {
```
Comment (lbwexler) [inline client-app/src/admin/tests/Select/SelectTestPanel.js]: these should be hoist components -- let's discuss why.

**Comment by lbwexler** [comment]
Peter -- marked this as WIP.  Let's take a look at this together on Monday -- think we can whip the code into shape pretty quickly.  Also, would like to have our tests refer to our options, (e.g. enableCreate, asyncFn) rather than the names of the underlying react-select classes.

**Comment by peterjcoffman** [comment]
@lbwexler 

I think I agree with you on all points. I was trying to be too lazy.

I've pushed a new version.

**Pull request merged**